### ISSUE TITLE
bug: postgresql unknown type character varying

### ISSUE THREAD

**Issue by reedlxm** [cloudquery/cloudquery#9141]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

as i want to sync from a postgresql to another postgresql,
take an error seams not support character varying type.

### Expected Behavior

i guess this is a bug or something goes wrong. that character varying is a very common type 

### CloudQuery (redacted) config

```
kind: source
spec:
  name: postgresql
  path: cloudquery/postgresql
  version: "v1.0.5"
  destinations: ["backup"]
  spec:
    connection_string: "postgres://"
---
kind: destination
spec:
  name: backup
  path: cloudquery/postgresql
  version: "v3.0.0"
  spec:
    connection_string: "postgres://"
```

### CloudQuery (redacted) logs

Starting migration with tables for: postgresql (v1.0.5) -> [backup (postgresql@v3.0.0)],
Error: failed to sync v1 source postgresql: failed to migrate source postgresql (v1.0.5) on destination backup (postgresql@v3.0.0) : failed to call Migrate: rpc error: code = Unknown desc = failed listing postgres tables: got unknown PostgreSQL type "character varying(150)" for column "" of table "" while trying to convert it to CloudQuery internal schema type. Supported PostgreSQL types are ["real" "macaddr[]" "inet" "macaddr" "inet[]" "float" "double precision" "text" "bytea" "uuid[]" "boolean" "bigint" "jsonb" "cidr[]" "bigint[]" "integer" "uuid" "text[]" "cidr" "timestamp"]

### CloudQuery version

docker leatest

### Pull request (optional)

- [ ] I can submit a pull request

**Comment by hermanschaaf** [comment]
Thanks for reporting this @reedlxm - I think we recently had a similar issue https://github.com/cloudquery/cloudquery/issues/8922, but that fix got released in v1.0.5 that you are already using. I will take a look at this 👀

**Comment by hermanschaaf** [comment]
I was able to reproduce the issue locally by doing the following:

```
cloudquery> create database source;
CREATE DATABASE
Time: 0.256s

cloudquery> \c source
You are now connected to database "source" as user "hermanschaaf"
Time: 0.014s

source> create table test_varchar3 (name varchar(150));
CREATE TABLE
Time: 0.019s

cloudquery> create database backup;
CREATE DATABASE
Time: 0.256s

cloudquery> \c backup
You are now connected to database "backup" as user "hermanschaaf"
Time: 0.014s

backup> create table test_varchar3 (name varchar(150));
CREATE TABLE
Time: 0.022s
```

Now running `cloudquery sync` to sync the table results in an error:

```
Loading spec(s) from pg.yml
Starting migration with 1 tables for: postgresql (v1.0.5) -> [backup (postgresql@v3.0.0)]
Error: failed to sync v1 source postgresql: failed to migrate source postgresql (v1.0.5) on destination backup (postgresql@v3.0.0) : failed to call Migrate: rpc error: code = Unknown desc = failed listing postgres tables: got unknown PostgreSQL type "character varying(150)" for column "name" of table "test_varchar3" while trying to convert it to CloudQuery internal schema type. Supported PostgreSQL types are ["integer" "uuid" "cidr" "cidr[]" "inet" "double precision" "bytea" "jsonb" "uuid[]" "macaddr" "bigint" "float" "real" "text" "macaddr[]" "boolean" "text[]" "inet[]" "bigint[]" "timestamp"]
```

It also doesn't work to skip migrations with `--no-migrate`:

```
 cloudquery sync --no-migrate pg.yml
Loading spec(s) from pg.yml
Starting sync for: postgresql (v1.0.5) -> [backup (postgresql@v3.0.0)]
Error: failed to sync v1 source postgresql: failed to write for postgresql (v1.0.5) -> backup (postgresql@v3.0.0): failed to CloseAndRecv client: rpc error: code = Internal desc = write failed: failed waiting for destination client got unknown PostgreSQL type "character varying(150)" for column "name" of table "test_varchar3" while trying to convert it to CloudQuery internal schema type. Supported PostgreSQL types are ["bigint" "macaddr[]" "inet[]" "uuid[]" "cidr[]" "macaddr" "boolean" "float" "real" "text" "text[]" "bigint[]" "integer" "double precision" "uuid" "cidr" "inet" "bytea" "jsonb" "timestamp"]
```

So my assumption is that this is happening because the destination table has already been created by some process external to CloudQuery? At the moment, the internal type system normalizes column types so column types on the destination may not be exactly the same as on the source (this is so that we can handle other destinations as well, but we are working on improving this).

I think we can allow this though by normalizing the types in our type system. I will try this and let you know how it goes
